### ISSUE TITLE
Unexpected Token

### ISSUE THREAD

**Issue by lpender** [taylorhakes/promise-polyfill#65]
Thanks for this!

When attempting to polyfill using:

```
import 'promise-polyfill/src/polyfill';
```

I get:

```
[14:48:13] Unexpected token (2:7) while parsing /path/to/src/node_modules/promise-polyfill/src/polyfill.js while parsing file: /path/to/src/node_modules/promise-polyfill/src/polyfill.js
[14:48:13] SyntaxError: Unexpected token (2:7) while parsing /path/to/src/node_modules/promise-polyfill/src/polyfill.js while parsing file: /path/to/src/node_modules/promise-polyfill/src/polyfill.js
    at DestroyableTransform.end [as _flush] (/path/to/src/node_modules/insert-module-globals/index.js:96:21)
    at DestroyableTransform.<anonymous> (/path/to/src/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:115:49)
    at DestroyableTransform.g (events.js:286:16)
    at emitNone (events.js:86:13)
    at DestroyableTransform.emit (events.js:185:7)
    at prefinish (/path/to/src/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:465:12)
    at finishMaybe (/path/to/src/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:473:7)
```

I'm using `gulp`, `watchify`, `babel` :)

I'm wondering if there's a dependency version mismatch ?

**Comment by taylorhakes** [comment]
This should be fixed in 7.0.1. Please let me know if it works

**Comment by sambshapiro** [comment]
I'm having a similar issue with `gulp`.
```
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^
Error: index.js from UglifyJs
SyntaxError: Unexpected token: name (Promise) [./~/promise-polyfill/src/polyfill.js:1,0]
```
